pytestqt/wait_signal.py

```diff
@@ -54,11 +57,76 @@ def _quit_loop_by_timeout(self):
             self._loop.quit()
 
     def _cleanup(self):
+        # store timeout message before the data to construct it is lost
+        self._timeout_message = self.get_timeout_error_message()
         if self._timer is not None:
             _silent_disconnect(self._timer.timeout, self._quit_loop_by_timeout)
             self._timer.stop()
             self._timer = None
 
+    def get_timeout_error_message(self):
+        pass
```

Comment:
I've changed it in the code, but coveralls still complains about that line not being covered. So, should I use a pragma? I don't see any configuration files in place yet.